### PR TITLE
acpi: add page

### PR DESCRIPTION
**Pull request by morevi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/acpi.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# acpi
+
+> Shows battery status or thermal information.
+
+- Show battery information:
+
+`acpi`
+
+- Show termal information:
```
Comment (agnivade) [inline pages/linux/acpi.md]: typo at thermal

**Comment by sbrl** [comment]
Thanks for the PR, @morevi!

Looks liek you haven't signed the CLA yet. The email address in the commits you've made needs to  be registered against your GitHub account for it to work.

**Comment by morevi** [comment]
I can't sign the CLA, the email address in the commits is registered on my account, i tried put to change the email from secondary to primary, and login in and out. I don't know what it could be.

**Comment by morevi** [comment]
I don't know if it has anything to do, but after fixing my system time (it was 2 hours in advance), it seems that I have been able to sign the CLA.

File: pages/linux/acpi.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show thermal information:
+
+`acpi -t`
+
+- Show cooling device information:
+
+`acpi -c`
+
+- Show thermal information using fahrenheit:
```
Comment (schneiderl) [inline pages/linux/acpi.md]:
```suggestion
- Show thermal information using Fahrenheit:
```

File: pages/linux/acpi.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show thermal information:
+
+`acpi -t`
+
+- Show cooling device information:
+
+`acpi -c`
+
+- Show thermal information using Fahrenheit:
```
Comment (sbrl) [inline pages/linux/acpi.md]: Perhaps this will read better?
```suggestion
- Show thermal information in fahrenheit:
```

Comment (agnivade) [reply pages/linux/acpi.md]: Shouldn't the F be in capital case ?

Comment (sbrl) [reply pages/linux/acpi.md]: Good catch, @agnivade - I've checked and that is indeed the case. I've updated my suggestion.

Comment (agnivade) [reply pages/linux/acpi.md]: Made the change myself :)

**Pull request merged**